app/uk/gov/hmrc/customs/declarations/information/services/ApiSubscriptionFieldsService.scala

```diff
@@ -56,4 +61,12 @@ trait ApiSubscriptionFieldsService {
     }
 
   }
+  
+  private def invalidAuthenticatedEori(authenticatedEori: Option[String]): Boolean = {
+    if (authenticatedEori.isEmpty || (authenticatedEori.isDefined && authenticatedEori.get.trim.isEmpty)) {
```

Comment:
try (authenticatedEori.isDefined && !authenticatedEori.get.trim.isEmpty) to flip to `valid....`

Reply:
Done.